modules/BuildTools.Tasks/RunBase.cs

```diff
@@ -65,15 +51,24 @@ public abstract class RunBase : Task
         /// </summary>
         public bool UseShellExecute { get; set; }
```

Comment:
npm publish uses MaxRetries and UseShellExecute, I assume because it's unreliable. Don't know why it's shell execute though.

Reply:
It uses shell execute because npm is actually npm.cmd. But this isn't an issue for ToolTask. It's already handled the complexity of launching/killing cmd.exe.